### PR TITLE
Feature/event service

### PR DESCRIPTION
**Pull request by zabila**

### CODE REVIEW

File: src/Sportalytics.Event.Domain/Entities/ApiSports/Fixture.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Sportalytics.Event.Domain.Entities.ApiSports;
+
+public class Fixture
+{
+    [JsonPropertyName("id")]
```
Comment (Puchkof) [inline src/Sportalytics.Event.Domain/Entities/ApiSports/Fixture.cs]: On JsonObject attribute with camelCase naming policy on class level everywhere

File: src/Sportalytics.Event.Infrastructure/Services/ApiSportsService/BackgroundApiSportsService.cs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+            var responseDto = new ProcessApiSportsResponseDto(apiResponse.Response);
+            var command = new ProcessSportEventsCommand(responseDto, stoppingToken);
+            await _sender.Send(command, stoppingToken);
+
+            await Task.Delay(_timeToNextRun, stoppingToken);
+        }
+    }
+
+    private async Task<JsonDocument?> DoRequest(HttpClient client, CancellationToken stoppingToken)
```
Comment (Puchkof) [inline src/Sportalytics.Event.Infrastructure/Services/ApiSportsService/BackgroundApiSportsService.cs]: SendRequestAsync

Comment (zabila) [reply src/Sportalytics.Event.Infrastructure/Services/ApiSportsService/BackgroundApiSportsService.cs]: Done)

File: src/Sportalytics.Event.Domain/Entities/ApiSports/Fixture.cs (hunk 1 above)
Comment (zabila) [reply src/Sportalytics.Event.Domain/Entities/ApiSports/Fixture.cs]: I want to have uppercase for naming.

**Review by Puchkof — APPROVED** [review]

**Pull request merged**